# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../esdl/make.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur esdl-0.94.1025_orig/Makefile esdl-0.94.1025/Makefile
--- esdl-0.94.1025_orig/Makefile	2004-10-25 11:27:07.000000000 +0200
+++ esdl-0.94.1025/Makefile	2005-04-22 18:58:45.578255408 +0200
@@ -41,7 +41,8 @@
 release: target relsrc relwin
 
 ## Install needs to find the erlang directory
-ERL_DIR := $(shell echo 'io:format("~s~n",[code:root_dir()]),halt().' | erl | sed 's,^[0-9]*> *,,g' | tail +2)
+#ERL_DIR := $(shell echo 'io:format("~s~n",[code:root_dir()]),halt().' | erl | sed 's,^[0-9]*> *,,g' | tail +2)
+ERL_DIR=/usr/lib/erlang/
 
 INSTALLDIR = $(ERL_DIR)/lib/$(ESDL_VSN)
 
diff -Naur esdl-0.94.1025_orig/c_src/Makefile esdl-0.94.1025/c_src/Makefile
--- esdl-0.94.1025_orig/c_src/Makefile	2004-10-25 11:27:07.000000000 +0200
+++ esdl-0.94.1025/c_src/Makefile	2005-04-22 18:48:01.371189768 +0200
@@ -18,7 +18,7 @@
 #GL_INCLUDE_DIR      = OpenGL
 #USE_GL_INCLUDE_DIR  = -DFULL_PATH_TO_GL_INCLUDE_DIR
 
-  GL_LIBS		= -L$(OGLDIR)/lib -lGL -lGLU 
+  GL_LIBS		= -L$(OGLDIR)/lib -lGL -lGLU -L/usr/X11R6/lib 
   GL_INCS		= -I$(OGLDIR)/include/$(GL_INCLUDE_DIR)
 
   SDL_LIBS		= $(shell sdl-config --libs)
